nextflow.config

```diff
@@ -88,7 +98,6 @@ profiles {
     }
     docker {
         docker.enabled         = true
-        docker.userEmulation   = true
```

Comment:
The `userEmulation` line was removed from the pipeline because it triggered the following warning:
`WARN: Undocumented setting docker.userEmulation is not supported anymore - please remove it from your config.`

Reply:
Thanks so much for the explanation. The removal makes sense 😄